index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Übung6</title>
+        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8">
+    </head>
+
+    <header>
+        <div class="container">
+            <a id="logo">
+                <img src="images/logo1.png" alt="logo">
+            </a>
+        </div>
+    </header>
+
+    <body>
+        <section id="personal">
+            <div>
+                <h2>Persönliches</h2>
+                <form method="POST" action="contact.php">

```

Comment:
Wenn du einen File upload in deiner Form hast musst du die Eigenschaft **enctype="multipart/form-data"** vergeben